dependency-reduced-pom.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.kainos.ea</groupId>
+  <artifactId>lc-kainosjobs-back</artifactId>

```

Comment:
got a feeling this was meant to be backend - just be careful here - this is the api side, so surely api makes more sense than -back

Reply:
changed to api